python/lsst/ts/externalscripts/make_LOVE_stress_tests.py

```diff
@@ -0,0 +1,299 @@
+                    if "category" not in msg or (
+                        "option" in msg and msg["option"] == "subscribe"
+                    ):
+                        continue
+                    self.received_messages = self.received_messages + 1
+                    tracing = msg["tracing"]
+                    tracing["client_rcv"] = Time.now().tai.datetime.timestamp()
+                    self.msg_traces.append(tracing)
+
+    async def subscribe_to(self, csc, salindex, stream, topic_type):
+        """Subscribes to the specified stream"""
+
+        subscribe_msg = {
+            "option": "subscribe",
+            "category": topic_type,
+            "csc": csc,
+            "salindex": salindex,
+            "stream": stream,
+        }
+        await self.websocket.send_str(json.dumps(subscribe_msg))
+
+    async def start_ws_client(self):
+        """Start client websocket connection"""
+
+        async with aiohttp.ClientSession() as session:
+            self.websocket = await session.ws_connect(self.websocket_url)
+            for csc in self.event_streams:
+                for stream in self.event_streams[csc]:
+                    await self.subscribe_to(
+                        csc.split(":")[0], csc.split(":")[1], stream, "event"
+                    )
+            for csc in self.telemetry_streams:
+                for stream in self.telemetry_streams[csc]:
+                    await self.subscribe_to(
+                        csc.split(":")[0], csc.split(":")[1], stream, "telemetry"
+                    )
+            await self.handle_message_reception()
+
+
+class StressLOVE(salobj.BaseScript):
+    """Run a stress test for one or more CSCs.
+
+    Notes
+    -----
+    **Details**
+
+    * Run a LOVE stress test by generating several client connections
+    that will listen to every event and telemetry of the specified CSCs
+    """
+
+    def __init__(self, index):
+        super().__init__(index=index, descr="Run a stress test for one or more CSCs")
+
+        # number of clients to simulate on the stress test
+        self.number_of_clients = 10
+
+        # list to store clients connections
+        self.clients = []
+
+        # commands timeout
+        self.cmd_timeout = 10
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/StressLOVE.yaml
+            title: StressLOVE v1
+            description: Configuration for StressLOVE
+            type: object
+            properties:
+              location:
+                description: Domain of the running LOVE instance to stress
+                type: string
+              username:
+                description: Username to authenticate connections
+                type: string
+              password:
+                description: Password to authenticate User
+                type: string
```

Comment:
username and password will be published to the EFD. I strongly suggest finding a safer way to deal with these secrets, such as using secrets in the Kubernetes deployment to set env vars.